### ISSUE TITLE
Custom track style doesn't center the marker

### ISSUE THREAD

**Issue by RyanMitchellWilson** [ptomasroos/react-native-multi-slider#39]
I'm trying to make a slider with a custom trackStyle but whenever I give it a different height, the marker doesn't adjust to the center. 
<img width="622" alt="screen shot 2017-09-28 at 11 49 00 am" src="https://user-images.githubusercontent.com/11371200/30981983-49a512c4-a443-11e7-81c9-05f1fbf26cc7.png">

My slider:
```
<Multislider
  trackStyle={{height: 15}}/>
```

**Comment by Kyonru** [comment]
This looks like the track start on the center of the maker and goes down...  A way to solve this is having a view with the same background color of the track to cover that empty area...

**Comment by ptomasroos** [comment]
Hopefully we can solve this easier when the new view engine is working on android and doesn't need to absolute position this.

**Comment by Kyonru** [comment]
By the moment making a custom maker and giving "top: 10" (change 10 for whatever that fit you) can be another solution.

**Comment by damikdk** [comment]
I use `marginTop` in markerStyle in my project, it looks good

**Comment by ptomasroos** [comment]
Right, this is because Android has not been supporting overflow, but this seems to be coming in RN 0.57 so hopefully that will allow us to make the rendering easier
